communication/youtube/youtube-guidelines.md

```diff
@@ -189,6 +189,9 @@ Below are a good and incorrect example:
 * Incorrect description: `Data Protection WG Bi-Weekly Meeting for 2/26/2020`
 * Correct description: `Data Protection WG Bi-Weekly Meeting for 20200226`
 
+A short video tutorial of the editing of descriptions and playlists can be found here:
+* https://drive.google.com/file/d/1_GwcKgnEhVlnm7w2HJKS3zIybXq5Bavj/view?usp=sharing
```

Comment:
We should probably put this up in YT 🤣 
what do you think?

Reply:
+1 what playlist should this be under? @castrojo should we create an admins playlist?